build/CoreGraphics/dll/CoreGraphics.def

```diff
@@ -233,6 +233,7 @@ LIBRARY CoreGraphics
         CGContextSetDirty
         CGContextGetBacking
         CGContextDrawGlyphRun
+        CGContextIsPointInPath
```

Comment:
It's an existing function, and will be removed with the CGD2D changes

---
In reply to: [88314147](https://github.com/Microsoft/WinObjC/pull/1398#discussion_r88314147) [](ancestors = 88314147)